### PR TITLE
Added serialization in pt create/delete and ptg create/delete operati…

### PR DESCRIPTION
**Pull request by ashutosh-mishra**
…ons.

### CODE REVIEW

**Comment by ashutosh-mishra** [comment]
@YogeshRajmane @rajendramachani @hemanthravi @njagadish

File: gbpservice/nfp/orchestrator/modules/service_orchestrator.py [hunk 1]
```diff
@@ -226,7 +227,16 @@ def _create_event(self, event_id, event_data=None,
             LOG.debug("poll event started for %s" % (ev.id))
             self._controller.poll_event(ev, max_times=10)
         else:
-            ev = self._controller.new_event(id=event_id, data=event_data)
+            if serialize:
+                LOG.info(_LI('in serialize, waiting for 15 secs'))
```
Comment (njagadish) [inline gbpservice/nfp/orchestrator/modules/service_orchestrator.py:231]: Need to have proper log text.

File: gbpservice/nfp/orchestrator/modules/service_orchestrator.py [hunk 2]
```diff
@@ -251,17 +262,21 @@ def network_function_device_notification(self, context, notification_data):
                 elif operation == 'delete':
                     event_id = self.rpc_event_mapping[resource][1]
                 elif operation == 'pt_add':
+                    serialize = True
```
Comment (njagadish) [inline gbpservice/nfp/orchestrator/modules/service_orchestrator.py:265]: - Isn't the issue with simultaneous member addition through heat template? If so, why are policy target operations serialized?
- Don't understand the consumer add operation also. Can we discuss?

**Comment by njagadish** [comment]
Merging this to run CI on the April 07 branch. Once the comments are discussed, this will be merged to main branch.

**Pull request merged**